kratos/utilities/mls_shape_functions_utility.h

```diff
@@ -37,6 +38,8 @@ class MLSShapeFunctionsUtility
 
 public:
 
+    using DenseSpace = UblasSpace<double, Matrix, Vector>;
```

Comment:
Seems unused here
Maybe move to cpp?

Or did you mean to use the ublas interface instead?

Reply:
I think I added it when trying different QR decompositions. Removing it.

Reply:
OFC, it is used in the cpp to instantiate the QR :facepalm: